js/about/brave.js

```diff
@@ -43,7 +43,7 @@ class AboutBrave extends React.Component {
       <div className='siteDetailsPageContent aboutAbout'>
         <div className='title'>
           <span className='sectionTitle' data-l10n-id='versionInformation' />
-          <span className='fa fa-clipboard' title='Copy password to clipboard' onClick={this.onCopy} />
```

Comment:
Something like this should work, but maybe another fix is needed for passing attributes to about pages, I'll merge this in the meantime either way though.
copyToClipboard.title=Copy to clipboard